ui/app/templates/vault/cluster/auth.hbs

```diff
@@ -17,6 +17,36 @@
     </EmptyState>
   </div>
 {{else}}
+  {{#if this.customMessages.bannerMessages.length}}
+    {{#each this.customMessages.bannerMessages as |bannerMessage|}}
+      <Hds::Alert @type="inline" @color="warning" data-test-alert={{bannerMessage.id}} as |A|>
```

Comment:
Just a thought: I wonder if the default color on these should be `info` instead, since we use many warning level banners so their custom messages may be lost, and it seems like most of the use cases we presume will be information rather than a warning

Reply:
@yongqikuang do you have thoughts on this?

Reply:
I'm going to update the banner color in a separate PR :)